src/aftership-card.ts

```diff
@@ -111,7 +111,7 @@ class AftershipCard extends LitElement {
                 </paper-item-body>
                 <paper-item-body class="last">
                   <div>
-                    ${item.last_checkpoint
+                    ${item.last_checkpoint.location
```

Comment:
I think you should check both
```suggestion
                    ${item.last_checkpoint && item.last_checkpoint.location
```

Reply:
You are right, my mistake